README.md

```diff
@@ -39,6 +39,7 @@ Help output:
       -v, --vertical       Display with vertical orientation [default].
       -x, --hex            Include hex color numbers on chart.
       -z, --horizontal     Display with horizontal orientation.
+      -m, --force-term=N   Skip termcap color detection and force N colors.
```

Comment:
This is useful on Windows under CONHOST or WindowsTerminal.exe because windows-curses currently reports 0 colors for `tigetnum('colors')`. However, it could be useful for probing color support in other places where the term settings might be confused (e.g. tmux or screen).